src/PBot/Issues/Pig.cs

```diff
@@ -61,7 +61,7 @@ public Pig()
                 await response.Send($"Alternatively, `{username}` may not be on any task forces. See https://github.com/Particular/Strategy/blob/master/definitions/taskforces.md");
             }
 
-            await response.Send(results.SelectMany(GetMessages).ToArray());
+            await response.Send(string.Join("\r\n", results.SelectMany(GetMessages)));
```

Comment:
I swear we tried that when wrote the pig me command in the first case and it did not work